catalog-entities/marketplace/plugins/azure-repositories.yaml

```diff
@@ -29,7 +29,9 @@ metadata:
 spec:
   author: Parfümerie Douglas
   publisher: Red Hat
-  support: tech-preview
+  support:
+    name: Parfümerie Douglas
```

Comment:
If the repo for the plugin is in the backstage community plugins, I think we should set this to "Backstage Community"